test/libsolidity/util/BytesUtils.h

```diff
@@ -32,67 +32,69 @@ namespace test
 class BytesUtils
```

Comment:
If all is static you might as well change the class to a namespace (this input was fed to me by @christianparpart )